packages/visual-editing/src/ui/context-menu/ContextMenu.tsx

```diff
@@ -55,22 +59,45 @@ function ContextMenuItem(props: {node: ContextMenuNode; onDismiss?: () => void})
         fontSize={1}
         icon={node.icon}
         padding={2}
+        // menu={{
```

Comment:
Can probably remove the commented menu prop?

Reply:
Good point, I'll add a comment because this will be needed when a `@sanity/ui` PR lands.